src/main/java/org/apache/commons/lang3/text/StrTokenizer.java

```diff
@@ -1075,45 +1048,45 @@ public String getContent() {
         return new String(chars);
     }
 
-    //-----------------------------------------------------------------------
     /**
      * Creates a new instance of this Tokenizer. The new instance is reset so
      * that it will be at the start of the token list.
-     * If a {@link CloneNotSupportedException} is caught, return <code>null</code>.
-     * 
+     * If a {@link CloneNotSupportedException} is caught, return {@code null}.
+     *
      * @return a new instance of this Tokenizer which has been reset.
      */
+    @Override
     public Object clone() {
         try {
             return cloneReset();
-        } catch (CloneNotSupportedException ex) {
+        } catch (final CloneNotSupportedException ex) {
             return null;
         }
     }
 
     /**
      * Creates a new instance of this Tokenizer. The new instance is reset so that
      * it will be at the start of the token list.
-     * 
+     *
      * @return a new instance of this Tokenizer which has been reset.
      * @throws CloneNotSupportedException if there is a problem cloning
      */
     Object cloneReset() throws CloneNotSupportedException {
         // this method exists to enable 100% test coverage
-        StrTokenizer cloned = (StrTokenizer) super.clone();
+        final StrTokenizer cloned = (StrTokenizer) super.clone();
         if (cloned.chars != null) {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It seems unusual to set the value of a variable only when it is already set. You might want to switch the null check of != on line 1077 to == check before setting the field value on line 1078 or verify if the check is needed at all.